### PR TITLE
Recipe for inotify-tools 3.14

### PR DESCRIPTION
**Pull request by keuv-grvl**

### CODE REVIEW

**Comment by dougalsutherland** [comment]
`./autogen.sh: line 3: aclocal: command not found` can probably be fixed by adding `automake` as a build requirement.

File: recipes/inotify-tools/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+{% set name = "inotify-tools" %}
+{% set version = "3.14" %}
+{% set sha256 = "97a4fdf9132da47c2baa677c1a427c435ee55a26df42b2b74a41489b9fbd2323" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
```
Comment (dougalsutherland) [inline recipes/inotify-tools/meta.yaml]: This line can be deleted.

File: recipes/inotify-tools/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  url: https://github.com/rvoicilas/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
```
Comment (dougalsutherland) [inline recipes/inotify-tools/meta.yaml]: `True` should be `true` here.

File: recipes/inotify-tools/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+requirements:
+  build:
+    - toolchain
+
+test:
+  commands:
+    - inotifywatch --help > /dev/null || [[ $? == 1 ]]
```
Comment (dougalsutherland) [inline recipes/inotify-tools/meta.yaml]: What is the `[[ $? == 1 ]]` accomplishing here? Doesn't `conda` fail if `$?` is any nonzero value, i.e. the same situations as the `||` branch would be taken?

File: recipes/inotify-tools/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+    - inotifywait --help > /dev/null || [[ $? == 1 ]]
+
+about:
+  home: https://github.com/rvoicilas/inotify-tools
+  license: GPL-2
+  license_family: GPL-2
```
Comment (dougalsutherland) [inline recipes/inotify-tools/meta.yaml]: `license_family` should be just `GPL` I think.

File: recipes/inotify-tools/meta.yaml (hunk 3 above)
Comment (dougalsutherland) [reply recipes/inotify-tools/meta.yaml]: Incidentally, this is actually what's causing the linter to complain, which is a bug in the linter (https://github.com/conda-forge/conda-smithy/issues/576). But I think you should probably drop it anyway. :)

Comment (keuv-grvl) [reply recipes/inotify-tools/meta.yaml]: Actually, `inotifywatch --help` returns `1` so I have no other way to test the `inotifywatch` execution (conda fails if one of the test commands returns a non zero status). I could test `which inotifywatch` but the program itself is not executed.

**Comment by dougalsutherland** [comment]
Whoops, I totally misread what the test was doing yesterday; sorry about that.

You could put it back, and just put a comment at the end of the line like `# --help returns 1` to both avoid the linter bug and show why you're doing that. :)

File: recipes/inotify-tools/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+    - libtool
+
+test:
+  commands:
+    - which inotifywatch
+    - which inotifywait
+
+about:
+  home: https://github.com/rvoicilas/inotify-tools
+  license: GPL-2
```
Comment (dougalsutherland) [inline recipes/inotify-tools/meta.yaml]: Whoops, this should actually be `GPL-2.0` for consistency.

File: recipes/inotify-tools/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+  license_family: GPL
+  license_file: COPYING
+  summary: 'This is a package of some commandline utilities relating to inotify.'
+  description: |
+    The general purpose of this package is to allow inotify's features to be
+    used from within shell scripts.  Read the man pages for further details.
```
Comment (dougalsutherland) [inline recipes/inotify-tools/meta.yaml]: I know you just copied this from the repo, but I don't know that "Read the man pages for more details" is really adding much to our description here. :)

**Comment by keuv-grvl** [comment]
The comment at the end of the test commands works well, thank you!

**Comment by dougalsutherland** [comment]
Thanks, and sorry for the delay @keuv-grvl!

**Comment by keuv-grvl** [comment]
Thanks for your help @dougalsutherland !

**Pull request merged**